<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Authentification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	 border-image: url(:/images/login.jpg);
}

QLabel{
		font-family: 'Monotype Corsiva',serif; 
		font-size: 18px;
        font-style: bold;
		color :white;
}

 QPushButton{width: 100px; height: 27px;
						color: white;
                        font-family: 'Calibri';font-size: 16px;
                        background-color:rgb(80,40,40); border: 1px solid black;border-radius: 6px;}
                        
QPushButton:pressed{width: 100px; height: 27px;
                        font-family: 'Calibri';font-size: 16px;
						color: white;
                        background-color: rgb(200,200,200); border: 1px solid black;border-radius: 6px;}

QPushButton:disabled{width: 100px; height: 27px;
                        font-family: 'Calibri';font-size: 16px;
						color: gray;
                        background-color: rgba(20,140,160,100); border: 1px solid black;border-radius: 6px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ipButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Adresse du serveur</string>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton{
background: rgba(255,255,255,0);
                        border-image: url(:/images/server.png);border-radius: 6px;}
                        
QPushButton:pressed{width: 100px; height: 30px; color: black;
                        font-family: 'Comic Sans MS';font-size: 16px;
                        background-color: rgb(128,128,255); border: 1px solid white;border-radius: 6px;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>37</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton{
background: rgba(255,255,255,0);
                        border-image: url(:/images/fermer.png);border-radius: 6px;}
                        
QPushButton:pressed{width: 100px; height: 30px; color: black;
                        font-family: 'Comic Sans MS';font-size: 16px;
                        background-color: rgb(128,128,255); border: 1px solid white;border-radius: 6px;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="verticalWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#verticalWidget
{
	background-color: rgba(0,0,0,160);
	border-radius: 15px;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="labelAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="pseudoLabel">
              <property name="text">
               <string>Nom utilsateur : </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="pseudoEdit">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
                height: 30px; 
		        border-radius: 5px;
				font-family: Consolas,serif; 
				font-size: 18px;
                font-style: bold;
				color : blue;
				 background: rgb(255,255,255);
}</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel">
              <property name="text">
               <string>Mot de passe :    </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordEdit">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
                height: 30px; 
		        border-radius: 5px;
				font-family: Consolas,serif; 
				font-size: 18px;
                font-style: bold;
				color : blue;
				 background: rgb(255,255,255);
}</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="loginButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Connexion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="mdpForgetButton">
          <property name="styleSheet">
           <string notr="true">#mdpForgetButton
{
	background-color: rgba(20,20,20,0);
	border: 1px solid  rgba(20,20,20,0);
	height: 15px;
	font-size: 12px;
}</string>
          </property>
          <property name="text">
           <string>Mot de passe oublié ? </string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="loginErrorLabel">
          <property name="styleSheet">
           <string notr="true">QLabel{
		font-family: calibri;
		background-color: rgba(255,0,0,150);
		font-size: 12px;
        font-style: bold;
		color : white;
		border-radius: 6px;
}</string>
          </property>
          <property name="text">
           <string>ERREUR: Nom d'utilisateur ou mot de passe incorrect. Veuillez réessayer!</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
